packages/google-cloud/package.json

```diff
@@ -103,6 +106,7 @@
     "@google-cloud/prediction": "^0.1.2",
     "@google-cloud/pubsub": "^0.2.0",
     "@google-cloud/resource": "^0.1.1",
+    "@google-cloud/speech": "^0.1.0",
```

Comment:
I think we might want to wait until the package is actually published before listing it as a dependency. Currently all the CI Jobs for this PR fail since `npm install` fails for this.

IMO we should do stuff in this order

* Merge Speech PR
* Release Speech API (v0.1.0)
* Add dependency for Speech on umbrella package
* Release umbrella package.